_includes/schemas/webhooks_request.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Webhook",
+  "description": "Webhooks allow you to subscribe to certain events",
+  "type": "object",
+  "properties": {
+    "id": { "type": "string", "description": "The unique identifier for a webhook. Will be generated by shipcloud." },
```

Comment:
Could you please enter a line break when defining more than one attribute? That'd be awesome.